.buildkite/scripts/steps/ess.sh

```diff
@@ -13,13 +13,6 @@ function ess_up() {
     return 1
   fi
 
-  export EC_API_KEY=$(retry -t 5 -- vault kv get -field=apiKey kv/ci-shared/platform-ingest/platform-ingest-ec-prod)
-  
-  if [[ -z "${EC_API_KEY}" ]]; then
```

Comment:
this check I should remain? maybe change the error message through. @v1v wdyt?

Reply:
There is no need, see https://github.com/elastic/elastic-agent/pull/8377#discussion_r2132172576